### PR TITLE
[DOCS-2766] Updates to Security Platform landing pages

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Updates all Security Platform landing pages to be more consistent with the current UI and add more context to each product area.

### Motivation
Clean-up prior to nav reorganization / new content

### Preview
https://docs-staging.datadoghq.com/sarina/security-landing-page/security_platform/
https://docs-staging.datadoghq.com/sarina/security-landing-page/security_platform/cloud_workload_security/
https://docs-staging.datadoghq.com/sarina/security-landing-page/security_platform/cspm/
https://docs-staging.datadoghq.com/sarina/security-landing-page/security_platform/security_monitoring/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cspm/_index.md [hunk 1]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
...
+- **Security posture score**: Percentage of your environment that satisfies all of your active [Datadog’s OOTB rules][1]. Formula: `(# of evaluation:pass findings) / (total # of findings)`. Datadog then weighs this formula by severity: low severity rules have a weighting of "1" and critical severity rules have a weighting of "5". This means critical severity rules impact scores five times more than low severity rules to put greater emphasis on the rules that pose greater security risk. The score is also normalized to treat all all resource types and resource volumes the same (for example, 500 failing containers are weighted the same as three failing S3 buckets in the computed score). This normalization factor allows scores to be comparable across your cloud accounts, without the risk they are heavily skewed if one account has more containers, or another has fewer storage buckets.
```
Comment (jtappa) [inline content/en/security_platform/cspm/_index.md:24]:
```suggestion
- **Security posture score**: Percentage of your environment that satisfies all of your active [Datadog OOTB rules][1]. Formula: `(# of evaluation:pass findings) / (total # of findings)`. Datadog then weighs this formula by severity: low severity rules have a weighting of "1" and critical severity rules have a weighting of "5". This means critical severity rules impact scores five times more than low severity rules to put greater emphasis on the rules that pose greater security risk. The score is also normalized to treat all all resource types and resource volumes the same (for example, 500 failing containers are weighted the same as three failing S3 buckets in the computed score). This normalization factor allows scores to be comparable across your cloud accounts, without the risk that they are heavily skewed if one account has more containers, or another has fewer storage buckets.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

File: content/en/security_platform/_index.md [hunk 2]
```diff
@@ -13,10 +24,39 @@ Cloud Security Posture Management is not currently available in US1-FED or US3.
...
+
+## Cloud Security Posture Management
+
+[Cloud Security Posture Management (CSPM)][4] tracks the security hygiene and compliance posture of your production environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks. See security posture scores across your infrastructure and trace each score back to the applicable benchmark or framework criteria.
+
+{{< img src="security_platform/cspm_overview.png" alt="Cloud Security Posture Management scores in Datadog" width="100%">}}
+
+## Cloud Workload Security
+
+[Cloud Workload Security (CWS)][5] monitors file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, and workloads, like Kubernetes clusters, in real time at the kernel level. Cloud Workload Security uses the unified Datadog Agent, so if you’re already using Datadog to monitor your environment, there’s no need to provision additional resources.
```
[the remark this answers is not in the document]
Comment (sarina-dd) [reply content/en/security_platform/_index.md:48]: Thanks for catching this! I'm changing it back to CWS as the team confirmed it's still CWS, just shows as `Workload Security` in app.

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 3]
```diff
@@ -1,20 +1,22 @@
 ---
-title: Cloud Workload Security
+title: Workload Security
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md:2]:
```suggestion
title: Cloud Workload Security
```

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 4]
```diff
@@ -1,20 +1,22 @@
...
 kind: documentation
 ---
 
 ## Overview
 
-Datadog Cloud Workload Security (CWS) detects threats to your production workloads in real-time. With Cloud Workload Security, monitor file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, docker containers, or Kubernetes clusters, in real-time at the kernel level. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories. Use **Process Execution Monitoring** to monitor process executions for suspicious, malicious, or anomalous activity.
+Datadog Workload Security detects threats to your production workloads in real-time. With Workload Security, monitor file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, docker containers, or Kubernetes clusters, in real-time at the kernel level. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories. Use **Process Execution Monitoring** to monitor process executions for suspicious, malicious, or anomalous activity.
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md]:
```suggestion
Datadog Cloud Workload Security detects threats to your production workloads in real-time. With Cloud Workload Security, monitor file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, docker containers, or Kubernetes clusters, in real-time at the kernel level. Use **File Integrity Monitoring (FIM)** to watch for changes to key files and directories. Use **Process Execution Monitoring** to monitor process executions for suspicious, malicious, or anomalous activity.
```

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 5]
```diff
@@ -1,20 +1,22 @@
...
 
-Cloud Workload Security uses the Datadog Agent, so if you’re already using Datadog to monitor your environment, there’s no need to provision additional resources or introduce new agents. If you don't already have the Datadog Agent set up, [start with setting up the Agent][1]. As a part of the Datadog platform, you can easily combine real-time threat detection with metrics, logs, traces, and other telemetry to see the full context surrounding a potential attack on your workloads.
+{{< img src="security_platform/cws/workload_security_rules.png" alt="Workload Security rules in the Datadog app" width="100%">}}
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md:10]:
```suggestion
{{< img src="security_platform/cws/workload_security_rules.png" alt="Cloud Workload Security rules in the Datadog app" width="100%">}}
```

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 6]
```diff
@@ -1,20 +1,22 @@
...
+
+Workload Security uses the Datadog Agent, so if you’re already using Datadog to monitor your environment, there’s no need to provision additional resources or introduce new agents. If you don't already have the Datadog Agent set up, [start with setting up the Agent][1]. As a part of the Datadog platform, you can easily combine real-time threat detection with metrics, logs, traces, and other telemetry to see the full context surrounding a potential attack on your workloads.
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md]:
```suggestion
Cloud Workload Security uses the Datadog Agent, so if you’re already using Datadog to monitor your environment, there’s no need to provision additional resources or introduce new agents. If you don't already have the Datadog Agent set up, [start with setting up the Agent][1]. As a part of the Datadog platform, you can easily combine real-time threat detection with metrics, logs, traces, and other telemetry to see the full context surrounding a potential attack on your workloads.
```

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 7]
```diff
@@ -1,20 +1,22 @@
...
 
 ## Get started
 
 {{< whatsnext >}}
   {{< nextlink href="/security_platform/cloud_workload_security/getting_started">}}Complete setup and configuration{{< /nextlink >}}
   {{< nextlink href="/security_platform/cloud_workload_security/workload_security_rules">}}Learn about Workload Security rules{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md:18]:
```suggestion
  {{< nextlink href="/security_platform/cloud_workload_security/workload_security_rules">}}Learn about Cloud Workload Security rules{{< /nextlink >}}
```

File: content/en/security_platform/cloud_workload_security/_index.md [hunk 8]
```diff
@@ -1,20 +1,22 @@
...
-  {{< nextlink href="/security_platform/default_rules/#cat-workload-security">}}Start using out-of-the-box Workload Security rules in your environment{{< /nextlink >}}
+  {{< nextlink href="/security_platform/default_rules/#cat-workload-security">}}Start using out-of-the-box Workload Security rules{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/cloud_workload_security/_index.md:19]:
```suggestion
  {{< nextlink href="/security_platform/default_rules/#cat-workload-security">}}Start using out-of-the-box Cloud Workload Security rules{{< /nextlink >}}
```

File: content/en/security_platform/cspm/_index.md [hunk 9]
```diff
@@ -1,14 +1,8 @@
 ---
-title: Cloud Security Posture Management
+title: Posture Management
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:2]:
```suggestion
title: Cloud Security Posture Management
```

File: content/en/security_platform/cspm/_index.md [hunk 10]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
 
 ## Overview
 
-Datadog Cloud Security Posture Management (CSPM) makes it easier to assess and visualize the current and historic security posture of your cloud environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.
+Datadog Posture Management makes it easier to assess and visualize the current and historic security posture of your cloud environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:14]:
```suggestion
Datadog Cloud Security Posture Management (CSPM) makes it easier to assess and visualize the current and historic security posture of your cloud environment, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.
```

File: content/en/security_platform/cspm/_index.md [hunk 11]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
...
 
 - **Resource**: A configurable entity that needs to be continuously scanned for adherence with one or more controls. Examples of AWS instance resources include hosts, containers, security groups, users, and customer-managed IAM policies.
 
-{{< img src="security_platform/cspm/getting_started/resource.png" alt="CSPM resource" style="width:65%;">}}
+{{< img src="security_platform/cspm/getting_started/resource.png" alt="Posture management resource information in the Datadog app" style="width:65%;">}}
 
 - **Rule**: A rule evaluates the configuration of a resource to validate an element related to one or more controls. Rules may map to multiple controls, requirements, and frameworks.
 
-{{< img src="security_platform/cspm/getting_started/rules.png" alt="Rules and frameworks" style="width:75%;">}}
+{{< img src="security_platform/cspm/getting_started/rules.png" alt="A list of Posture Management rules" style="width:65%;">}}
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:36]:
```suggestion
{{< img src="security_platform/cspm/getting_started/rules.png" alt="A list of Cloud Security Posture Management rules" style="width:65%;">}}
```

File: content/en/security_platform/cspm/_index.md [hunk 12]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
...
 
 - **Findings**: A finding is the primary primitive for a rule evaluation against a resource. Every time a resource is evaluated against a rule, a finding is generated with a Pass or Fail status.
 
 - **Framework**: A collection of requirements that map to an industry benchmark or regulatory standard.
 
-{{< img src="security_platform/cspm/getting_started/frameworks.png" alt="Frameworks overview" style="width:100%;">}}
+{{< img src="security_platform/cspm/getting_started/frameworks.png" alt="The frameworks overview in the Posture Management landing page" style="width:100%;">}}
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:42]:
```suggestion
{{< img src="security_platform/cspm/getting_started/frameworks.png" alt="The frameworks overview in the Cloud Security Posture Management landing page" style="width:100%;">}}
```

File: content/en/security_platform/cspm/_index.md [hunk 13]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
...
 
 ## Get started
 
 {{< whatsnext >}}
-  {{< nextlink href="/security_platform/cspm/getting_started">}}Configure your cloud environment for CSPM scanning.{{< /nextlink >}}
-  {{< nextlink href="/security_platform/cspm/findings">}} Search and explore findings to reveal the details about a misconfigured resource, enabling you to identify the steps needed to fix it.{{< /nextlink >}}
-  {{< nextlink href="/security_platform/cspm/frameworks_and_benchmarks">}}Get details about Datadog’s OOTB configuration rules to compare your configuration to applicable regulatory standards, controls, and industry benchmarks.{{< /nextlink >}}
+  {{< nextlink href="/security_platform/cspm/getting_started">}}Complete setup and configuration{{< /nextlink >}}
+  {{< nextlink href="/security_platform/default_rules/#cat-cloud-configuration">}}Implement out-of-the-box Posture Management rules{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:48]:
```suggestion
  {{< nextlink href="/security_platform/default_rules/#cat-cloud-configuration">}}Implement out-of-the-box Cloud Security Posture Management rules{{< /nextlink >}}
```

File: content/en/security_platform/cspm/_index.md [hunk 14]
```diff
@@ -17,45 +11,45 @@ Cloud Security Posture Management is not currently available in this site.
...
+  {{< nextlink href="/security_platform/cspm/findings">}} Learn about Posture Management findings{{< /nextlink >}}
```
Comment (sarina-dd) [inline content/en/security_platform/cspm/_index.md:49]:
```suggestion
  {{< nextlink href="/security_platform/cspm/findings">}} Learn about Cloud Security Posture Management findings{{< /nextlink >}}
```

**Review by sarina-dd — COMMENTED** [review]
Updated instances of WS and PM

File: content/en/security_platform/security_monitoring/_index.md [hunk 15]
```diff
@@ -5,24 +5,22 @@ kind: documentation
...
-  {{< nextlink href="/security_platform/detection_rules">}}Learn about creating custom detection rules{{< /nextlink >}}
+  {{< nextlink href="/security_platform/security_monitoring/getting_started">}}Complete setup and configuration{{< /nextlink >}}
+  {{< nextlink href="/security_platform/default_rules#cat-log-detection">}}Start using out-of-the-box Security Monitoring rules{{< /nextlink >}}
+  {{< nextlink href="/security_platform/detection_rules">}}Create your own custom detection rules{{< /nextlink >}}
 {{< /whatsnext >}}
 
-[1]: https://app.datadoghq.com/security
-[2]: https://app.datadoghq.com/security/configuration/rules
-[3]: https://app.datadoghq.com/security/configuration/rules/new
+[1]: /security_platform/security_monitoring/sucurity_home/
```
Comment (sarina-dd) [inline content/en/security_platform/security_monitoring/_index.md:24]:
```suggestion
[1]: /security_platform/security_monitoring/security_home/
```

**Review by jtappa — APPROVED** [review]

**Pull request merged**